src/review/caption/Caption.style.tsx

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components'
+
+import { Button } from '../../button'
+import { normalizeHorizontally } from '../../layout/layoutNormalizer'
+
+export const StyledCaption = styled.div`
+  display: flex;
+  ${normalizeHorizontally};
+
+  > * {
+    flex: 1 0 auto;
+  }
+
+  > * + * {
```

Comment:
Maybe create a "StyledButton" to be more explicit, even though it's for one simple extra rule